src/main/java/reposense/model/StandaloneAuthor.java

```diff
@@ -0,0 +1,32 @@
+package reposense.model;
+
+/**
+ * Represents an {@code StandaloneAuthor} in {@code StandaloneConfig}.
+ */
+public class StandaloneAuthor {
```

Comment:
`StandaloneAuthor` represents a `StandaloneAuthor`???
Can probably just mention `an author` instead, don't need to directly reference any class.